client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/ContactDetails.tsx

```diff
@@ -46,23 +59,16 @@ const ContactDetails = (props: Props) => {
             <Typography variant='body2'>
                 <b>שם משפחה:</b> {interactedContact.lastName}
             </Typography>
-            <Typography variant='body2'>
-                <b>תאריך המגע:</b>{' '}
-                {format(new Date(interactedContact.contactDate), 'dd/MM/yyyy')}
-            </Typography>
-            {interactedContact.contactType && (
-                <Typography variant='body2'>
-                    <b>סוג מגע:</b>{' '}
-                    {
-                        contactTypes.get(+interactedContact.contactType)
-                            ?.displayName
-                    }
-                </Typography>
-            )}
-            {interactedContact.extraInfo && (
-                <Typography variant='body2'>
-                    <b>פירוט אופי המגע:</b> {interactedContact.extraInfo}
-                </Typography>
+            {interactedContact.contactEvents && (
+                <Tooltip title={tooltipText} placement='top' arrow>
+                    <Typography variant='body2'>
+                        <b>סוג מגע:</b>{' '}
```

Comment:
{' '}, what is this? blank space?

Reply:
so it would say 'סוג מגע: הדוק' and not 'סוג מגע:הדוק'